docs/bond-rest-api-spec.yaml

```diff
@@ -0,0 +1,1357 @@
+                        S&P: BBB-
+                      ]
+                    coupon_rate: 15
+                    coupon_type: Floating
+                    coupon_frequency: Quarterly
+                    first_coupon_date: 1333170000
+                    maturity_date: 1641880800
+                    first_settlement_date: 1325397600
+                    amount_outstanding: 2250000000
+                    face_value: 1000000000
+                head: 65cd3a3ce088b265b626f704b7f3db97b6f12e848dccb35d7806f3d0324c71b709ed360d602b8b658b94695374717e3bdb4b76f77886953777d5d008558247dd
+                link: https://bond-rest-api/bonds?head=65cd3a3ce088b265b626f704b7f3db97b6f12e848dccb35d7806f3d0324c71b709ed360d602b8b658b94695374717e3bdb4b76f77886953777d5d008558247dd
+        '400':
+          $ref: '#/components/responses/BadRequestError'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /bonds/{bond_id}:
+    get:
+      description: Returns the bond with specified ID
+      parameters:
+        - in: path
+          name: bond_id
+          schema:
+            type: string
+          required: true
+          description: ISIN of the bond
+          example: US00206RDA77
+        - $ref: '#/components/parameters/head'
+      responses:
+        '200':
+          description: Success response with requested bond
+          content:
+            'application/json':
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/BondModel'
+                  head:
+                    $ref: '#/components/schemas/HeadModel'
+                  link:
+                    $ref: '#/components/schemas/LinkModel'
+              example:
+                data:
+                  bond_id: US00206RDA77
+                  issuing_organization_id: T
+                  corporate_debt_ratings: [
+                      Fitch: BBB,
+                      Moodys: Ba3,
+                      S&P: BBB-
+                    ]
+                  coupon_rate: 15
+                  coupon_type: Floating
+                  coupon_frequency: Quarterly
+                  first_coupon_date: 1333170000
+                  maturity_date: 1641880800
+                  first_settlement_date: 1325397600
+                  amount_outstanding: 2250000000
+                  face_value: 1000000000
+                head: 65cd3a3ce088b265b626f704b7f3db97b6f12e848dccb35d7806f3d0324c71b709ed360d602b8b658b94695374717e3bdb4b76f77886953777d5d008558247dd
+                link: https://bond-rest-api/bonds/US00206RDA77?head=65cd3a3ce088b265b626f704b7f3db97b6f12e848dccb35d7806f3d0324c71b709ed360d602b8b658b94695374717e3bdb4b76f77886953777d5d008558247dd
+        '400':
+          $ref: '#/components/responses/BadRequestError'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /holdings:
```

Comment:
Maybe I misunderstood what you said today in stand up, but is there an endpoint to delete a holding?

Reply:
I think he had meant that this action would be taken care of through organizations?

Reply:
If its to be handle through organization, I think it should be added to the example and the end point description of that the holdings can be deleted through there.  Though I think, as this is a delete operation, we might want a endpoint `DELETE organization/{organization_id}/{asset_id}`

Reply:
Based on the discussions from last week, it seemed like we were not implementing the delete functionality since it was not implemented in the original application. Based on the discussion today it seems like we might need to implement that after all.

Reply:
Today during standup, I mentioned that there wasn't currently a `GET` route for holdings, since that resource is fetched as part of the `GET /organizations` route

Reply:
I think we had decided to not implement the delete organization functionality, since delete holding is still in the transaction protos. It was only deleting an organization that we had taken out.